zerver/openapi/zulip.yaml

```diff
@@ -7071,6 +7093,24 @@ paths:
                     the server has a database index for unread messages.
 
                     **Changes**: See changelog at [Construct a narrow](/api/construct-narrow).
+
+                    In Zulip 9.0 (feature level 263), narrows gained support for a
+                    new filter: `is:followed`, which returns messages from topics that the user has
+                    [followed](/help/follow-a-topic).
+
+                    In Zulip 9.0 (feature level 250), narrows gained support
+                    for two new filters related to stream messages: `channel` and `channels`;
+                    which are aliases for (and return the same results as) the `stream` and
+                    `streams` filters respectively.
+
+                    In Zulip 9.0 (feature level 249), narrows gained support for a new
+                    filter, `has:reaction`, which returns messages with at least one
+                    [emoji reaction](/help/emoji-reactions).
+
+                    In Zulip 7.0 (feature level 177), narrows gained support
+                    for three new filters related to direct messages: `is:dm`, `dm` and
+                    `dm-including`; replacing and deprecating `is:private`, `pm-with` and
+                    `group-pm-with` respectively.
```

Comment:
These changes look like a misresolved merge conflict. It's clear that we intend for this documentation to have moved, see `**Changes**: See changelog at [Construct a narrow](/api/construct-narrow).`

In general, whenever you see a merge conflict, you'll want to use `git log -p <path>` to read the change that your work conflicted with and understand how to make your changes correctly on top of those. Doing such a workflow should make this sort of failure mode where you accidentally undo someone else's work very unlikely.